dockerfiles/ci/centos/6/php-5.6/Dockerfile

```diff
@@ -0,0 +1,72 @@
+FROM datadog/dd-trace-ci:centos-6 as base
```

Comment:
This file for [5.6](https://raw.githubusercontent.com/DataDog/dd-trace-php/f0589aad7996a10397593a3592cd714a99982975/dockerfiles/ci/centos/6/php-5.6/Dockerfile) is identical to the ones for [5.5](https://raw.githubusercontent.com/DataDog/dd-trace-php/f0589aad7996a10397593a3592cd714a99982975/dockerfiles/ci/centos/6/php-5.5/Dockerfile) and [5.4](https://raw.githubusercontent.com/DataDog/dd-trace-php/f0589aad7996a10397593a3592cd714a99982975/dockerfiles/ci/centos/6/php-5.4/Dockerfile)

Reply:
Yep! Very good point, I'll make separate Dockerfiles for the PHP major versions.